ws-nextjs-app/pages/[service]/new_live/[id]/live.test.tsx

```diff
@@ -35,6 +37,18 @@ describe('Live Page', () => {
     ).toBeInTheDocument();
   });
 
+  it('should render the posts', async () => {
+    await act(async () => {
+      render(<Live pageData={mockPageData} />);
+    });
+
+    expect(screen.getAllByText('Breaking news')[0]).toBeInTheDocument();
```

Comment:
**nitpick** getByText will return the first result if there are multiples -

`screen.getAllByText('Breaking news')[0]` is the same as `screen.getByText('Breaking news')`


```suggestion
    expect(screen.getByText('Breaking news')).toBeInTheDocument();
```

Reply:
Oops, I was wrong - apologies for this, but thanks for trying it out and for checking!